src/cascade/input_data/db/bundle.py

```diff
@@ -0,0 +1,327 @@
+import logging
+
+import pandas as pd
+
+from cascade.core.db import cursor, connection
+
+CODELOG = logging.getLogger(__name__)
+
+# FIXME: There is a shared function that get's the official mapping, I think. Or an sql query at least.
+MEASURES = {6: "incidence", 9: "mtexcess"}
+
+
+def _bundle_is_frozen(execution_context):
+    """Checks if data for the current model_version_id already exists in tier 3.
+    """
+
+    model_version_id = execution_context.parameters.model_version_id
+    query = f"""
+    select exists(
+             select * from epi.t3_model_version_dismod
+             where model_version_id = {model_version_id}
+    )
+    """
+    with cursor(execution_context) as c:
+        c.execute(query)
+        exists = c.fetchone()[0]
+
+    return exists == 1
+
+
+def _get_bundle_id(execution_context):
+    """Gets the bundle id associated with the current model_version_id.
+    """
+    model_version_id = execution_context.parameters.model_version_id
+
+    query = f"""
+    SELECT bundle_id
+    FROM epi.model_version
+    WHERE model_version_id = %(model_version_id)s
+    """
+
+    with cursor(execution_context) as c:
+        CODELOG.debug(f"Looking up bundle_id for model_version_id {model_version_id}")
+        c.execute(query, args={"model_version_id": model_version_id})
+        bundle_ids = list(c)
+
+        if not bundle_ids:
+            raise ValueError(f"No bundle_id associated with model_version_id {model_version_id}")
+
+        if len(bundle_ids) > 1:
+            raise ValueError(f"Multiple bundle_ids associated with model_version_id {model_version_id}")
+
+        return bundle_ids[0][0]
+
+
+def _get_bundle_data(execution_context, bundle_id, tier=3):
+    """Downloads the tier 2 data for the bundle associated with the current model_version_id.
+    """
```

Comment:
should this be "tier 2 or tier 3 data" ?

Reply:
Yep